src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -52,10 +52,12 @@ object Conseil extends App with LazyLogging with EnableCORSDirectives {
   }
 
   val bindingFuture = Http().bindAndHandle(route, conseil_hostname, conseil_port)
-  logger.info(s"Bonjour..")
-  while(true){}
-  bindingFuture
-    .flatMap(_.unbind())
-    .onComplete(_ => system.terminate())
+  logger.info("Bonjour...")
 
+  sys.addShutdownHook {
```

Comment:
I'm guessing this is a safer way to run Conseil indefinitely?

Reply:
actually it's a safe way to terminate the additional threads on which the Http binding and the actor-system exist

Conseil would continue running even when the main thread exits (with no need to loop indefinitely), because the threads spawned to handle the server is `non-daemon`, which means that it will outlive the thread that spawned it

Reply:
Awesome, thanks for the clarification!